docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -298,32 +306,50 @@ Let's start with the WordPress pod:
 TIP: In K9s you can open a shell in a pod by selecting that pod and then hitting the kbd:[S] key.
 
 [start=2]
-. Replace the files inside the pod with the ones from the restore:
+. Replace the files inside the pod with the ones from the restore. 
+  Using rsync here simplifies things by only copying files that are different from the target and also deleting any additional malicious files an attacker may have added:
 
 ....
-$ cd /var/www/html # or whatever value has `mountPath`
-$ mv data /
-$ rm -r *
-$ cp -R /data/wordpress-pvc/* .
+# apt update && apt install rsync
+# rsync -a --delete /restore/ /var/www/html/
 ....
 
 [start=3]
 . Since it is a PHP application, nothing else is needed for this pod.
 
-In the case of the MariaDB pod, the application must be restarted after restoring the files:
+=== Restoring the database dump
+
+In the case of the MariaDB pod, this backup annotation in the MariaDB deployment instructed K8up to create a database dump as text and store it in MinIO:
+
+[source,yaml]
+....
+spec:
+  template:
+    metadata:
+      annotations:
+        k8up.syn.tools/backupcommand: appuio.ch/backupcommand: /bin/bash -c 'mysqldump -uroot -p"${MARIADB_ROOT_PASSWORD}" --all-databases'
+....
+
+Thus the restoration procedure is different. 
+Instead of restoring files to a PVC, we can instead connect to the running database in the pod and restore directly from the dump.
 
 [start=4]
-. Follow the same logic as previously:
+. Get the snapshot ID for the database dump and save it to your local machine as `backup.sql`:
 
 ....
-$ cd /var/lib/mysql # Attention! Different path!
-$ mv data /
-$ rm -r *
-$ cp -R /data/mariadb-pvc/* .
+$ export SNAPSHOT_ID=$(restic snapshots --json --last --path /default-mariadb | jq -r '.[0].id')
+$ restic dump "${SNAPSHOT_ID}" /default-mariadb > backup.sql
```

Comment:
We could probably pipe this directly to a `restic dump... | kubectl exec...` thus making this a oneliner and remove the whole copy stuff below.

Reply:
I need to expand `$MARIADB_ROOT_PASSWORD` from the env inside the container for that. But just appending it to kubectl will expand it on the host system. I couldn't figure out if it's possible to escape it so it gets expanded inside?

What I came up with is this little adventure:

```bash
restic dump "${SNAPSHOT_ID}" /default-mariadb | kubectl exec -i "$MARIADB_POD" -- mysql -uroot --password=`kubectl exec "$MARIADB_POD" -- printenv "MARIADB_ROOT_PASSWORD"`
```

Not elegant but it works. Is there a better way?

Reply:
You could wrap it with `sh -c ...`
There's an example here: https://k8up.io/k8up/1.1.0/how-tos/prebackuppod.html

Reply:
Ah, alright! Thanks, changed it to that.